app/views/collections/show.rb

```diff
@@ -102,6 +103,21 @@ def version
 
     private
 
+    def og_image
+      og_image_from_promo || og_image_from_hero
+    end
+
+    def og_image_from_promo
+      return unless @landing_page.settings_layout_type == 'browse'
+      promo = @landing_page.promotions.find_by(position: 0)
+      promo && promo.file.present? ? promo.file.url : nil
+    end
+
+    def og_image_from_hero
+      hero_conf = hero_config(@landing_page.hero_image)
+      URI.join(root_url, hero_conf[:hero_image]) unless hero_conf.nil? || hero_conf[:hero_image].blank?
```

Comment:
actually the original reason was probably implemented as a fallback for when the hero image file was empty, so we were returning the root_url, which kind of doesn't make sense since that won't be an image anyway.
Now with the other checks and the hero_config code we have I think we'll not need to do this fallback.